.github/ISSUE_TEMPLATE.md

```diff
@@ -1,16 +1,17 @@
+---
+
 ### Environment summary
 
 **Install Method:** How did you install the CLI? (e.g. pip, interactive script, apt-get, Docker, MSI, nightly)  
 Answer here:
 
-**CLI Version:** What version of the CLI and modules are installed? (Available with `az --version`)  
+**CLI Version:** What version of the CLI and modules are installed? (The full output of `az --version`)  
```

Comment:
I've slightly changed the wording here.
Would still prefer more info that less when debugging.